api/android/build-android-lib.sh

```diff
@@ -404,6 +425,13 @@ if [[ $enable_snpe == "yes" ]]; then
     cp $SNPE_DIRECTORY/lib/dsp/libsnpe*.so api/src/main/jni/snpe/lib/ext/arm64-v8a
 fi
 
+# Update PyTorch option
+if [[ $enable_pytorch == "yes" ]]; then
+    sed -i "s|ENABLE_PYTORCH := false|ENABLE_PYTORCH := true|" api/src/main/jni/Android.mk
+    sed -i "s|PYTORCH_VERSION := 1.8.0|PYTORCH_VERSION := $pytorch_ver|" api/src/main/jni/Android-tensorflow-lite.mk
```

Comment:
Fix this line to Android-pytorch.mk, not tf-lite mk file.

Reply:
It's fixed. Thanks!